api/app/tests/hfi/test_hfi.py

```diff
@@ -260,3 +260,18 @@ def test_valid_date_range_default_for_end_date_before():
     result = validate_date_range(DateRange(start_date=start_date, end_date=end_date))
     assert result.start_date.isoformat() == '2020-05-21'
     assert result.end_date.isoformat() == '2020-05-21'
+
+
+def test_inclusive_date_math():
```

Comment:
Can you please also add a test for `startDate` and `endDate` are on the same day (single day prep cycle) and that `days_in_range()` returns 1?